cadc-data-ops-fits/build.gradle

```diff
@@ -8,19 +8,27 @@ plugins {
 repositories {
     mavenCentral()
     mavenLocal()
+
+    // Remove when cadc-soda-server:1.2.1 is on mavenCentral()
```

Comment:
What is this? That version is on mavenCentral and since the first mavenCentral version was 1.2.0 this should never have been published to jcenter.